.github/publish-maven.sh

```diff
@@ -62,6 +62,7 @@ cat <<EOF >~/.m2/settings.xml
         <activeByDefault>true</activeByDefault>
       </activation>
       <properties>
+        <altSnapshotDeploymentRepository>snapshots::default::https://s01.oss.sonatype.org/content/repositories/snapshots</altSnapshotDeploymentRepository>
```

Comment:
Does this need a `altDeploymentRepository`, as well?

Reply:
We only had a alt for snapshot. Actually, I don't know why we had this settings.